### PR TITLE
Action

### PR DESCRIPTION
**Pull request by Gauthier-Lhomme**

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
Nice one !

File: src/components/ActionPage.jsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+          </P>
+          <DivCenter>
+            <ButtonFlash>Rendez-vous flash</ButtonFlash>
+          </DivCenter>
+        </DivCards>
+      </DivInfos>
+      <H1>Notre équipe</H1>
+      <CardFlex>
+        <Card>
+          <ImgContainer src={geffroy} alt="#" />
```
Comment (LoicBrassart) [inline src/components/ActionPage.jsx]: Add a *meaningful* value to the `alt` attribute ("Mr Geffroy", for example): it should describe the content of your picture

File: src/components/ActionPage.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            <DivLogo>
+              <a href="https://www.linkedin.com/in/favart/ ">
+                <ImgLogo src={linkedin} alt="logo linkedin" />
+              </a>
+            </DivLogo>
+          </Container>
+        </Card>
+
+        <Card>
+          <ImgContainer src={marsac} alt="#" />
```
Comment (LoicBrassart) [inline src/components/ActionPage.jsx]: Add a meaningful value to the alt attribute ("Mr Marsac", for example): it should describe the content of your picture

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- Useless to add licence files for every font
- The `Fonts` folder should be named `fonts`
- Some alts to change

**Review by Thoms-C — APPROVED** [review]
Noice !

**Review by LoicBrassart — APPROVED** [review]
Much better !

**Pull request merged**